Dockerfile

```diff
@@ -19,7 +19,12 @@ RUN apt-get update &&\
     apt-get install -y --no-install-recommends gnupg &&\
     curl -sL https://deb.nodesource.com/setup_12.x | bash - &&\
     apt-get update &&\
-    apt-get install -y --no-install-recommends nodejs jq
+    apt-get install -y --no-install-recommends nodejs jq docker docker-compose
```

Comment:
Question: do we need docker here? I'm pretty sure we can install docker-compose in standalone and have it connect to docker on the host system (by mounting `/var/run/docker.sock`)

Reply:
done